### DISCUSSION TITLE
Database Connection Error: SQLITE_CANTOPEN undefined:undefined

### DISCUSSION THREAD

**Discussion by sl5net** [requarks/wiki#3488] [Help / Questions]
**Question**
tried install it, as easy and fast as possible (sqlite).

**Host
OS: kUbuntu 20.10
Wiki.js version: Wiki.js 2.5.170
Database engine: sqlite (not installed first but later)

```
wget https://github.com/Requarks/wiki/releases/download/2.5.170/wiki-js.tar.gz
mkdir wiki
tar xzf wiki-js.tar.gz -C ./wiki
cd ./wiki
mv config.sample.yml config.yml
nano config.yml
npm rebuild sqlite3
node server
```
i changed to sqlite in the config. (`nano config.yml`) 
==>
```
Loading configuration from ./config.yml... FAILED
end of the stream or a document separator is expected at line 12, column 1:
    port: 3000
    ^
>>> Unable to read configuration file! Did you create the config.yml file?
```
First error.
Found that nano automatically writes before first line:

```
npm rebuild sqlite3
node server
```
please offer a one-liner-command for installation. will be great.

next error:
```
info: Using database driver sqlite3 for sqlite [ OK ]
2020-11-17T08:07:19.602Z [MASTER] info: Connecting to database...
2020-11-17T08:07:19.605Z [MASTER] error: Database Connection Error: SQLITE_CANTOPEN undefined:undefined
2020-11-17T08:07:19.605Z [MASTER] warn: Will retry in 3 seconds... [Attempt 1 of 10]
```

i tried an did then:

```
sudo apt-get install sqlite3 libsqlite3-dev
sqlite3 wiki.db
node server
```

but last error not changed.
what i need to do next?

**Comment by NGPixel** [comment]
Make sure the wiki.js node process has write access to the sqlite location on disk?

**Comment by danielehrhardt** [comment]
```
2020-12-16T16:54:29.058Z [MASTER] [31merror[39m: Database Connection Error: SQLITE_CANTOPEN undefined:undefined
2020-12-16T16:54:29.058Z [MASTER] [33mwarn[39m: Will retry in 3 seconds... [Attempt 4 of 10]
```
Same Problem with Docker

**Comment by FlipEnergy** [comment]
Make sure you set `DB_FILEPATH` to the expected location of your database file. For example `/data/wikijs.db`.

**Comment by lawrenceching** [comment]
In my case, I unexpectedly mounted a root folder into container
Therefore, wiki.js was unable to write DB file under the mounted folder.
